src/wallet/wallet.h

```diff
@@ -836,6 +836,12 @@ class ZTXOSelector {
     bool SelectsOrchard() const;
 };
 
+enum class RecipientType {
```

Comment:
Document this. In particular, note that `WalletExternalAddress` does not just mean the BIP 44 notion of "addresses used for external operations", but also "addresses we have potentially shown to users" (and thus includes legacy change addresses even though they are derived from the BIP 44 internal chain, c/f UA change addresses which we explicitly keep private).

Reply:
I would argue that unless "external" and "internal" here mean the same as they do elsewhere (in particular, in ZIPs 32 and 316), then these constants should be called something else.

Reply:
Fixed in https://github.com/zcash/zcash/pull/5715/commits/151074cbb13455afb8257354c0d856a79da3f332